WordPress/src/main/java/org/wordpress/android/ui/domains/management/composable/DomainsSearchTextField.kt

```diff
@@ -33,14 +37,25 @@ fun DomainsSearchTextField(
         onValueChange = onValueChange,
         enabled = enabled,
         placeholder = { Text(stringResource(placeholder)) },
-        shape = RoundedCornerShape(50),
+        shape = RoundedCornerShape(4.dp),
         leadingIcon = {
             Icon(
                 painter = painterResource(id = R.drawable.ic_search_white_24dp),
                 contentDescription = "",
                 tint = MaterialTheme.colorScheme.outline,
             )
         },
+        trailingIcon = {
+            if (value.isNotEmpty()) {
+                IconButton(onClick = { onValueChange("") }) {
+                    Icon(
+                        painter = painterResource(id = R.drawable.ic_cross_in_circle_white_24dp),
+                        contentDescription = "",
```

Comment:
I was wondering if it makes sense to add a `contentDescription` (e.g. "Clear search").

Reply:
Good idea! Resolved w/ https://github.com/wordpress-mobile/WordPress-Android/pull/19633/commits/d14ef5fbf61a009cce928743499ff669684b3c68

Reply:
Thank you Matthew 🚀